lib/openingScreen.dart

```diff
@@ -0,0 +1,111 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import 'menu.dart';
+
+class openingScreen extends StatefulWidget {
+  @override
+  State<openingScreen> createState() => _openingScreenState();
+}
+
+class _openingScreenState extends State<openingScreen> with TickerProviderStateMixin{
+  late AnimationController controller;
+  @override
+  void initState() {
+
+    controller=AnimationController(vsync: this,
+    duration: const Duration(seconds: 1),
+
+    )..addListener(() {
+      setState(() {
```

Comment:
You are calling setState with no reason.
There is nothing to update inside, why so?
It will just keep rebuilding the state of the app, and it is not a good idea.